recipes/empymod/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "empymod" %}
+{% set version = "1.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dba7b900b2fe0e811d8cae47cb53d81a05e851e1e9b21cecc37bb3d77da42cf5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  preserve_egg_dir: True
```

Comment:
Just commenting to make sure you actually need this.

Reply:
If I leave it out, does conda-forge get the right version with `setuptools_scm`?